src/main/java/tconstruct/library/weaponry/AmmoItem.java

```diff
@@ -76,35 +90,41 @@ public float getAmmoModifier() {
     }
 
     public boolean pickupAmmo(ItemStack stack, ItemStack candidate, EntityPlayer player) {
-        if (stack.getItem() == null || !stack.hasTagCompound() || !(stack.getItem() instanceof IAmmo)) return false;
+        if (stack.getItem() == null || !stack.hasTagCompound() || !(stack.getItem() instanceof IAmmo pickedup))
+            return false;
```

Comment:
Could you add brackets around this statement, since it's on a new line?
```suggestion
        if (stack.getItem() == null || !stack.hasTagCompound() || !(stack.getItem() instanceof IAmmo pickedup)) {
            return false;
        }
```